packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   We updated the @author in the typings generator from `Mendix UI Content Team` to `Mendix Widgets Framework Team`.
 
+### added
```

Comment:
`Added` should start with a capital.

Reply:
I think its possible to derive a changelog validator workflow from the release script to avoid this issue from happening again